### DISCUSSION TITLE
Use two IP adresses for Authelia behind NGINX (authelia.conf)

### DISCUSSION THREAD

**Discussion by 3DJupp** [authelia/authelia#5564] [Q&A] [answered]
Hey, i wanted to ask, if its possible to add a second IP in the authelia.conf, like i did in the auth_example_com.conf:
```
upstream authelia-ifaces {
    server 172.16.0.1:9091 fail_timeout=5s max_fails=3;
    server 172.16.0.2:9091 backup;
}
```

Within the authelia.conf (responsible for the /authelia backend on protected domains) there is this setting:
**set $upstream_authelia http://172.17.0.20:9091/api/verify;**

Unfortunately the upstream block seems not to be allowed:
**2023/06/15 09:12:59 [emerg] 1288#1288: "upstream" directive is not allowed here in /etc/nginx/snippets/authelia.conf:39**

Thank you 😄

**Comment by james-d-elliott** [comment, marked as answer]
Sorry for not seeing and answering this earlier.

I believe what you'd have to do here is create an internal server which listens on :9091 or similar with the upstream as you configured it, and then set `set $upstream_atuhelia http://127.0.0.1:9091/api/verify` where :9091 is the port the server listens on.

I'll mark as the answer but if it doesn't work let me know.
